tools/wptserve/wptserve/handlers.py

```diff
@@ -95,15 +95,29 @@ def list_items(self, base_path, path):
             link = urljoin(base_path, "..")
             yield ("""<li class="dir"><a href="%(link)s">%(name)s</a></li>""" %
                    {"link": link, "name": ".."})
+        items = []
+        last_item = None
```

Comment:
Can we call this `prev_item` instead? At first I thought it was "last" as in "last in the list".

Reply:
We totally can, but is that really all the feedback?

Reply:
Did you want more feedback? :p Apart from that it seems reasonable to land the change afaict.

Reply:
I was expecting my Python to not be up to snuff.